src/components/Questions/QuestionMultiple.vue

```diff
@@ -57,6 +61,16 @@
 					@keydown.enter.exact.prevent="onKeydownEnter">
 					{{ answer.text }}
 				</NcCheckboxRadioSwitch>
+				<NcCheckboxRadioSwitch v-if="isShowOtherAnswer"
+					:checked.sync="questionValues"
+					:value="valueOtherAnswer"
+					:name="`${id}-answer`"
+					:type="isUnique ? 'radio' : 'checkbox'"
+					:required="checkRequired('other-answer')"
+					@update:checked="onChange"
+					@keydown.enter.exact.prevent="onKeydownEnter">
+					{{ t('forms', 'Other:') }}&nbsp;<NcInputField placeholder="answer..." :value.sync="inputOtherAnswer" class="question__input" />
```

Comment:
```suggestion
					<NcInputField placeholder={{ t('forms', 'Other...') }} :value.sync="inputOtherAnswer" class="question__input" />
```

Please use the text directly as the placeholder. This is also the way Google handles it in Google Forms.

And please adjust the styling of the Input box to matcht the width of the other question types.

Last, as you also use two different placeholders for view/edit/submit mode, we should just move the placeholder text to AnswerTypes.js like for the other question types.

Reply:
> Thanks for the contribution @AIlkiv @Chartman123 we cannot use placeholders to deliver important information because of accessibility problems.
> 
> I think that this text field that is shown when other is selected should appear below the radio button and span the whole width of the container.

@marcoambrosini Thanks for your input 🙂 

Here are two screenshots how Google handles it currently:

|Edit|View|
|---|---|
|![image](https://github.com/nextcloud/forms/assets/16020496/1e1e8c67-bcd9-42f8-a114-bb454ab3e992)|![image](https://github.com/nextcloud/forms/assets/16020496/82f63b92-196e-4b13-94fb-69e9483ee549)|

So it's a combination of what I suggested and @AIlkiv implemented. As they don't have a border around the input box it looks fine with the input box in line with the radio/checkbox button.

How would Allkiv's version look like with a RTL language?

Reply:
> Please use the text directly as the placeholder. This is also the way Google handles it in Google Forms.
> 
> And please adjust the styling of the Input box to match the width of the other question types.
> 
> Last, as you also use two different placeholders for view/edit/submit mode, we should just move the placeholder text to AnswerTypes.js like for the other question types.
@Chartman123 

![image](https://github.com/nextcloud/forms/assets/5858915/16bc2a3d-a8a8-44a1-bc04-2e9b59fb3e05)

![image](https://github.com/nextcloud/forms/assets/5858915/7292347e-d398-4e4b-a08a-88b0083fe5c5)

![image](https://github.com/nextcloud/forms/assets/5858915/2404163f-45ce-4e4a-af05-c63d9519524c)

> How would Allkiv's version look like with a RTL language?

![image](https://github.com/nextcloud/forms/assets/5858915/50fe703d-4a66-47e3-bd36-695e4aff640c)

When editing, the layout of some fields is displayed incorrectly, but this is the topic of a separate task:

![image](https://github.com/nextcloud/forms/assets/5858915/f6b6991c-edbe-4bf4-bff7-4ef47248ea73)

Reply:
> When editing, the layout of some fields is displayed incorrectly, but this is the topic of a separate task:

Yes, there's already another PR open to improve RTL support :)